core/src/channel/voice_buffer.rs

```diff
@@ -28,13 +28,17 @@ impl DerefMut for GroupVoice {
 pub struct VoiceBuffer {
     id_counter: usize,
     buffer: VecDeque<GroupVoice>,
+    pub damper: bool,
```

Comment:
This doesn't have to be public

Reply:
Oh I see why you made it public, technically it's better to make a function for "pub fn set_damper(&mut self, damper: bool)" and then do the cleanup there.

Basically, you can move the code on line 132 into the set_damper function if you set it to false.

Reply:
Right, this sounds like a better approach to keep the value private too. Will change.